_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -59,6 +73,30 @@ class HealthCareApp extends React.Component {
     this.state = initialState;
   }
 
+  getNextUrl() {
+    const routes = this.props.route.childRoutes;
+    const panels = [];
+    let currentPath = this.props.location.pathname;
+    let nextPath = '';
+
+    if (currentPath === '/') {
+      currentPath = '/introduction';
+    }
+
+    for (let i = 0; i < routes.length; i++) {
```

Comment:
For loops...since Javascript is a lisp and ES6 added sensible functionality, we can do cute little functional programming things (and almost never write for loops) :)

https://tonicdev.com/56cbfb723a0cd70c00742642/56d5e89ec14e920d0064d25e